jest.config.js

```diff
@@ -9,6 +9,7 @@ module.exports = {
   coverageDirectory: '.artifacts/coverage/',
   snapshotSerializers: ['enzyme-to-json/serializer'],
   moduleNameMapper: {
+    '^@original-emotion/styled': '<rootDir>/node_modules/@emotion/styled',
```

Comment:
I don't think this is needed - I believe the errors were because of an import from `app/styled` instead of `@emotion/styled`